parkx_kode/model/Parkingplace.py

```diff
@@ -2,17 +2,18 @@
 
 
 class Parkingplace:
-    def __init__(self, id, name, address, zip_code, number_of_places, price_pr_hour, picture, details, available):
+    def __init__(self, id, name, address, zip_code, number_of_places, price_pr_hour, picture, details):
         self.id = id
         self.name = name
         self.address = address
         self.zip_code = zip_code
-        self.number_of_places = number_of_places
-        self.price_pr_hour = price_pr_hour
+        self.number_of_places = int(number_of_places)
+        self.price_pr_hour = int(price_pr_hour)
         self.picture = picture
         self.details = details
-        self.available = available
-        self.parkingStarted = datetime.datetime.now().strftime("%H:%M:%S")
+        self.available = True
+        # self.parkingStarted = datetime.datetime.now().strftime("%H:%M:%S")
```

Comment:
skal denne være kommentert ut?

Reply:
skal fjerne den fullstendig. takk for du ser over